### ISSUE TITLE
Button should be highlighted if `aria-expanded="true"`

### ISSUE THREAD

**Issue by huyenltnguyen** [freeCodeCamp/ui#12]
## Describe the bug

The Button component in `ui-components` should be highlighted if `aria-expanded="true"`. This will enable the migration of native `button` to ui-components `Button`.

However, this change will impact the display of `Dropdown` so we need to revisit the UI of this component as well.

| Dropdown with `aria-expanded="false"` | Dropdown with `aria-expanded="true"` |
| --- | --- |
| <img width="611" alt="Screenshot 2023-12-20 at 16 57 01" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/25715018/c0c69237-33d2-4a49-a173-d679b463ecc7"> | <img width="615" alt="Screenshot 2023-12-20 at 16 56 45" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/25715018/29aca961-fec8-409f-954a-ac4e339f24bd"> |

## Additional context

Related to freeCodeCamp/freeCodeCamp#52641.
